### ISSUE TITLE
If you already have your private keys and don´t know what to do with them or have not been able to get your funds back, check this out

### ISSUE THREAD

**Issue by nyhfreude** [langerhans/dogecoin-wallet-new#74]
This is the only solution i could find , any wallet allowed me to see my balance.
Remember that every address has a different has a different private key.

https://www.reddit.com/r/dogecoin/comments/7sb80p/eli5_how_to_send_coins_using_coinbin/

**Comment by langerhans** [comment]
This is an issue tracker and not a discussion forum.
